content/karpenter/index.md

```diff
@@ -68,10 +68,15 @@ The following example shows how to avoid provisioning large Graviton instances.
       'c6g.16xlarge'
 ```
 
-### Install the AWS Node Termination Handler when using Spot
+### Enable Interruption handling when using Spot
 
-At present, Karpenter does not handle the Spot Interruption Termination Notice (ITN) two-minute warning. In lieu of this, you can install [AWS Node Termination Handler](https://github.com/aws/aws-node-termination-handler) to gracefully cordon and drain your spot nodes when they are interrupted.
- Pods that require checkpointing or other forms of graceful draining, requiring the 2-mins before shutdown, will need NTH.
+As of Karpenter v0.19.3 there is [native node termination handling](https://karpenter.sh/v0.21.1/concepts/deprovisioning/#interruption) baked into Karpenter that watches for upcoming involuntary interruption events that would cause disruption to your workloads such as;
```

Comment:
```suggestion
Starting in v0.19.3, Karpenter supports [native interruption handling](https://karpenter.sh/preview/concepts/deprovisioning/#interruption), enabled through the `aws.interrupionQueue` value in [Karpenter settings](https://karpenter.sh/preview/concepts/settings/#configmap). Interruption handling will watch for upcoming involuntary interruption events that would cause disruption to your workloads such as:
```